packages/native/types/index.d.ts

```diff
@@ -0,0 +1,67 @@
+// Definitions by: Ben James <https://github.com/benhjames>
+// TypeScript Version: 2.8
+
+import {
+  CreateStyled as BaseCreateStyled,
+  CreateStyledComponentExtrinsic
+} from '@emotion/styled-base'
```

Comment:
This should not depend on `@emotion/styled-base` since package.json of this package does not have one.
(And please do not add the package, since it will add additional external dependencies)

Reply:
What's the recommendation in this case, since those dependencies shouldn't be added? Is copying and pasting those types from `@emotion/styled-base` an OK strategy?

Reply:
@derekstavis Although one adds duplicated codes, that would be a better approach.